src/main/java/aurora/objects/DukeException.java

```diff
@@ -0,0 +1,29 @@
+package aurora.objects;
+
+/**
+ * The DukeException class represents exceptions specific to this application.
+ */
+public class DukeException extends Exception {
+
+    /** Message of the exception. */
+    private String exceptionMessage;
+
+    /**
+     * Constructor of the DukeException class.
+     *
+     * @param exceptionMessage Message of the exception that describes it.
+     */
+    public DukeException(String exceptionMessage) {
+        super(exceptionMessage);
+        this.exceptionMessage = exceptionMessage;
+    }
+
+    /**
+     * Getter for the exception message.
+     *
+     * @return Message that describes the exception.
+     */
+    public String getExceptionMessage() {
```

Comment:
I am not sure if you need an exception message for this class specifically if you already call the parent constructor with the same message, since you can get it with `Exception::getMessage`.